### PR TITLE
[protoc] Automatically remove trailing spaces in generated pb.d.ts files

### PR DESCRIPTION
**Pull request by jankeromnes**
~~WIP: Untested. Please advise on how to run the generator. 😇~~

### CODE REVIEW

**Comment by aledbf** [comment]
@jankeromnes `leeway run components:generate-code-from-protobuf`

**Comment by jankeromnes** [comment]
@aledbf Please take a look. Seems to do what's expected, but there are a few non-whitespace changes since the last generation.

File: components/ws-daemon-api/go/mock/mock.go [hunk 1]
```diff
@@ -124,9 +124,6 @@ func (mr *MockWorkspaceContentServiceClientMockRecorder) WaitForInit(arg0, arg1
 type MockWorkspaceContentServiceServer struct {
 	ctrl     *gomock.Controller
 	recorder *MockWorkspaceContentServiceServerMockRecorder
-
-	api.UnimplementedInWorkspaceServiceServer
```
Comment (aledbf) [inline components/ws-daemon-api/go/mock/mock.go:128]: Please rollback this change. This code needs to be added manually after the code generator

**Comment by aledbf** [comment]
@jankeromnes besides my comment this change LGTM

**Comment by jankeromnes** [comment]
@aledbf Thanks! Added back the two lines. Please ✅ if this can be merged.

**Review by aledbf — APPROVED** [review]

**Pull request merged**